OriginateTheme/Swift/Models/Colors.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Colors.swift
+//  OriginateTheme
+//
+//  Created by Robert Weindl on 2017-02-14.
+//  Copyright © 2017 Originate Inc. All rights reserved.
+//
+
+import UIKit
+
+public struct Colors {
+    var dictionary: ThemeDefinition
+    
+    public var text: UIColor {
```

Comment:
Hi Allen yes this is just an example as the other comment above. Should demonstrate how the generated file should look like.